### ISSUE TITLE
Migrate to PyInstaller binary distribution

### ISSUE THREAD

**Issue by Moonlight1211** [Moonlight1211/clip#2]
Currently, clip only uses Linux-centric shell scripts to install and run the program, which is insufficient for package distribution and end-user accessibility in the software ecosystem. A better approach is to use a distribution tool to create a native executable binary of the application.

Current plan is to implement PyInstaller support, which may enable formal Windows support in the future.

**Comment by Moonlight1211** [comment]
Closed by pr #4
